src/pages/AskFeedback.js

```diff
@@ -46,6 +46,11 @@ const renderSelect = ({
           {option.text}
         </option>
       ))}
+      {label === 'circle' && (
+        <option key="none_1337" value="none">
```

Comment:
key attributes are only needed for JSX elements created inside a loop (via map ors something), without this this becomes a one-liner and the parenthesis are also removed by Prettier.
And can you rename the value of the <option> to "no_circle", I think this adds clarity especially in the render where this value is checked.